From 522f79e4ed58ea821dd939daa856a2d9dfd6f934 Mon Sep 17 00:00:00 2001
From: Neui <neuisen@googlemail.com>
Date: Sun, 2 May 2021 01:08:37 +0200
Subject: [PATCH] Use Gdk.Cursor.new_from_name()

Because GDK4 dropped new_for_display in favour of new_from_name.
---
 GTG/gtk/editor/taskview.py                | 8 ++++----
 GTG/plugins/unmaintained/tomboy/tomboy.py | 4 ++--
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/GTG/gtk/editor/taskview.py b/GTG/gtk/editor/taskview.py
index 4b510499..1313ead8 100644
--- a/GTG/gtk/editor/taskview.py
+++ b/GTG/gtk/editor/taskview.py
@@ -597,8 +597,8 @@ class TaskView(Gtk.TextView):
         tags = view.get_iter_at_location(x, y)[1].get_tags()
 
         # Reset cursor and hover states
-        cursor = Gdk.Cursor.new_for_display(window.get_display(),
-                                            Gdk.CursorType.XTERM)
+        cursor = Gdk.Cursor.new_from_name(window.get_display(),
+                                          'text')
         window.set_cursor(cursor)
 
         if self.hovered_tag:
@@ -613,8 +613,8 @@ class TaskView(Gtk.TextView):
         try:
             tag = tags[0]
             tag.set_hover()
-            cursor = Gdk.Cursor.new_for_display(window.get_display(),
-                                                Gdk.CursorType.HAND2)
+            cursor = Gdk.Cursor.new_from_name(window.get_display(),
+                                              'pointer')
             window.set_cursor(cursor)
             self.hovered_tag = tag
 
diff --git a/GTG/plugins/unmaintained/tomboy/tomboy.py b/GTG/plugins/unmaintained/tomboy/tomboy.py
index 28c4226a..d81cd58c 100644
--- a/GTG/plugins/unmaintained/tomboy/tomboy.py
+++ b/GTG/plugins/unmaintained/tomboy/tomboy.py
@@ -337,8 +337,8 @@ class TomboyPlugin():
         # cursor changes to a hand
 
         def realize_callback(widget):
-            cursor = Gdk.Cursor.new_for_display(eventbox.window.get_display(),
-                                                Gdk.CursorType.HAND2)
+            cursor = Gdk.Cursor.new_from_name(eventbox.window.get_display(),
+                                              'pointer')
             eventbox.window.set_cursor(cursor)
         eventbox.connect("realize", realize_callback)
         return eventbox
-- 
2.30.0

